### ISSUE TITLE
Immunity to all damage after dying as a possessed mob, repossessing, then becoming human again

### ISSUE THREAD

**Issue by Barerock** [Ladysnake/Dissolution#46]
So I have a massive tower about 300-400 blocks tall/deep with a giant pit that used to be in the middle of it (I built stuff in it and over it aka. the tower and portal stuff.) and I had some issues with falling. I fell a number of times and had to build back up every time like... 50 so blocks, but I'd get pushed off my nerd pole every once in a while and I compulsively switched on creative mode and forgot every time (It's a convenience thing, I hate walking super far to get items) and my possessed mob dies (doesn't matter which kind, although I've only done this in zombies so far I think). The bug started with zombies not doing any damage, and I just started a betweenlands game just to see what it's like and I got attacked and died and I couldn't possess anything (no undead nearby) after that NOTHING will give me damage. Not poison, drowning, falling, mobs, arrows. Nothing... and so I uninstalled Dissolution temporarily to see if I could fix it (I got back to humanity by forcing a weak soul btw) But it didn't fix anything. Now I just lack an awesome mod, and retain the problems I think it caused. This is the only mod I can think of that I installed before the problems started occurring. 
Thanks, great mod, bit buggy.

**Comment by Pyrofab** [comment]
First thought: this bug should be fixed by entering and leaving soul mode. `/dis cmode set ...` can help you with that.

**Comment by Barerock** [comment]
Not quite. The first instance was strange. I died while in creative as a possessed mob after falling some 30-40 blocks and the game crashed. Before I fell, a fairy started purifying me, and when I restarted the world I was human again, but at the same point as where I died. I think this first instance is what caused it, although in my betweenlands world, all that happened was that I needed to reset my soul strength in order to get back into humanity. I'll try that command out and see how it works. 
Thanks!

**Comment by Barerock** [comment]
Alright, so I tried that command and it functioned, but I still take zero damage.

**Comment by Barerock** [comment]
Yup. Just did that after fixing some texture stuff. Worked like a charm! Thanks for the help!
